src/org/zaproxy/zap/extension/ascan/ActiveScanAPI.java

```diff
@@ -128,7 +131,7 @@
 	public ActiveScanAPI (ExtensionActiveScan controller) {
 		this.controller = controller;
         this.addApiAction(new ApiAction(ACTION_SCAN,
-        		new String[] {PARAM_URL}, 
```

Comment:
Indentation/formatting changes should be removed. [1]

[1] https://github.com/zaproxy/zaproxy/wiki/DevGuidelines